packages/uiweb/src/lib/components/chat/ChatViewBubble/ChatViewBubble.tsx

```diff
@@ -339,15 +338,15 @@ const TwitterCard = ({
 export const ChatViewBubble = ({ decryptedMessagePayload }: { decryptedMessagePayload: IMessagePayload }) => {
   const { account } = useChatData();
   const position =
-    pCAIP10ToWallet(decryptedMessagePayload.fromDID).toLowerCase() !== account?.toLowerCase()
+    pCAIP10ToWallet(decryptedMessagePayload.fromCAIP10).toLowerCase() !== account?.toLowerCase()
       ? 0
       : 1;
   const { tweetId, messageType }: TwitterFeedReturnType = checkTwitterUrl({
     message: decryptedMessagePayload?.messageContent,
   });
   const [isGroup, setIsGroup] = useState<boolean>(false);
   useEffect(() => {
-    if (decryptedMessagePayload.toDID.split(':')[0] === 'eip155') {
+    if ((decryptedMessagePayload.toCAIP10).split(':')[0] === 'eip155') {
```

Comment:
I think there can be some edge case if the user has an nft profile on push

Reply:
yes for nft profile we will need alot of changes, we can do it in a different pr. What do you think?

Reply:
I think we should create a helper function which basically checks whether it is nft or group or user and gets the id accordingly but yeah will be better to be done in a separate pr